package/gluon-core/check_site.lua

```diff
@@ -70,6 +70,12 @@ if need_table({'dns'}, nil, false) then
 	need_number({'dns', 'cacheentries'}, false)
 end
 
+if need_table('next_node', nil, false) then
+	need_string_match('next_node.ip6', '^[%x:]+$', false)
+	need_string_match('next_node.ip4', '^%d+.%d+.%d+.%d+$', false)
+	need_string_array('next_node.name', false)
+end
```

Comment:
You have duplicated two lines from below. The `need_string_array` is not adjusted to the new check_site functions (and maybe we should validate the domain names at least a bit). The `need_table` is redundant now.

Reply:
yes it isn't this was done before the merge of the multi-domain stuff. It is adjusted now.

Reply:
validation is not yet done... not sure what would be a good way. There is this https://stackoverflow.com/questions/106179/regular-expression-to-match-dns-hostname-or-ip-address which certainly seems like overkill.

Reply:
Lua patterns don't support operators on groups anyways, so something like `^[a-z0-9%-%.]+$` is probably the best we can easily do.